src/dex/dex.cpp

```diff
@@ -0,0 +1,22 @@
+/******************************************************************************
+ * Copyright © 2013-2019 The Komodo Platform Developers.                      *
+ *                                                                            *
+ * See the AUTHORS, DEVELOPER-AGREEMENT and LICENSE files at                  *
+ * the top-level directory of this distribution for the individual copyright  *
+ * holder information and the developer policies on copyright and licensing.  *
+ *                                                                            *
+ * Unless otherwise agreed in a custom licensing agreement, no part of the    *
+ * Komodo Platform software, including this file may be copied, modified,     *
+ * propagated or distributed except according to the terms contained in the   *
+ * LICENSE file                                                               *
+ *                                                                            *
+ * Removal or modification of this copyright notice is prohibited.            *
+ *                                                                            *
+ ******************************************************************************/
+
+#include "dex.hpp"
+
+namespace antara
+{
+
```

Comment:
The pull request seem's imcomplete with empty files

Reply:
As the scope of this PR is just the order manager, I may leave the CEX and DEX classes empty. Then, a later PR can add their functionality.

Reply:
I would prefer not merge empty file and code that we are not using if it's possible it's will decrease our coverage once @ca333 will enable it

Reply:
I think that this is an inevitability sometimes; otherwise, I would need to write strictly in a bottom-up fashion. This way, when someone comes along to write the DEX class, they know what what it's called and which methods need to be implemented.

Reply:
The alternatives would be to either write the CEX and DEX first, or have half-empty implementations in the order manager functions.

Reply:
I've changed them to throw `not_implemented` exceptions.